DllImportGenerator/DllImportGenerator/GeneratorDiagnostics.cs

```diff
@@ -55,7 +55,7 @@ public class Ids
             public const string ConfigurationNotSupported = Prefix + "002";
         }
 
-        private const string Category = "DllImportGenerator";
+        private const string Category = "Usage";
```

Comment:
Maybe `SourceGeneration`? I don't think `Usage` is correct for the generator diagnostics.